src/math/math_utils.hh

```diff
@@ -49,31 +49,32 @@ namespace rascal {
   }
 
   namespace math {
-
-    // Reminder: C++ floating-point literals are automatically of type double
-    /// Pi to more digits than anyone could possibly need
-    const double PI = 3.14159265358979323846264338327950288419716939937510;
-    const double SQRT_PI =
-        1.7724538509055160272981674833411451827975494561223871282;
-    const double SQRT_TWO = std::sqrt(2.0);
-    const double INV_SQRT_TWO = std::sqrt(0.5);
-    const double SQRT_THREE = std::sqrt(3.0);
+    /// π to more digits than anyone could possibly need
+    constexpr double PI = 3.14159265358979323846264338327950288;
```

Comment:
Why use `constexpr` instead of `const`? Is there a good reason besides making me have to think harder about all the different ways to qualify variables in C++?

Reply:
`constexpr` means that the value is (1) const and (2) known at compile time. This means the compiler is able to do constant folding; which it might not do with `std::sqrt(3.0)`, since the value is created at compile-time.

Reply:
But is there a difference if it's just a plain old numeric literal, as it is here?

Reply:
This should not be different when considering optimization, but shows intent better.

And talking about qualifiers explosions, these should actually be `inline constexpr` (https://stackoverflow.com/a/54467111)

Reply:
actually, they can't be inline constexpr, since that's a c++17 extension ¯\_(ツ)_/¯